num/src/posit.rs

```diff
@@ -0,0 +1,160 @@
+// Rust language amplification library providing multiple generic trait
+// implementations, type wrappers, derive macros and other language enhancements
+//
+// Written in 2014 by
+//     Andrew Poelstra <apoelstra@wpsoftware.net>
+// Refactored & fixed in 2021 by
+//     Dr. Maxim Orlovsky <orlovsky@pandoracore.com>
```

Comment:
Please put your name here instead of these two

Reply:
done